histogram.js

```diff
@@ -118,14 +120,32 @@ class HistogramElement extends HTMLElement {
         this._histo.update(d);
       });
     }
-    else {
-      (async () => {
-        const data = await getDataFromAttr(this);
-        if (data) {
-          this._histo.update(data);
+
+    const renderHistogramWhenVisible = async () => {
+      if (!useBroker) {
+          try {
+              const data = await getDataFromAttr(this);
```

Comment:
Can you fix the indentation here? It should be 2 spaces. We should use 2 space in general for web apps..

Reply:
I fixed it! Please recheck it.